pkg/userdata/flatcar/provider.go

```diff
@@ -203,6 +205,17 @@ systemd:
             Environment=ALL_PROXY={{ .HTTPProxy }}
 {{- end }}
 
+    - name: containerd.service
+      dropins:
+        - name: 10-custom.conf
+          contents: |
+            [Service]
+            Restart=always
+            EnvironmentFile=-/etc/environment
+            Environment=CONTAINERD_CONFIG=/etc/containerd/config.toml
```

Comment:
Where are you getting this file from? it doesn't seem that you create it anywhere